samples/bluetooth/mesh/sensor_server/prj.conf

```diff
@@ -65,6 +65,12 @@ CONFIG_BT_MESH_SENSOR_SRV=y
 CONFIG_BT_MESH_SENSOR_SRV_SENSORS_MAX=5
 CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE=n
 
+# Increase default delayable message config to avoid insuffisient resources
+# during normal sample operation
+CONFIG_BT_MESH_ACCESS_DELAYABLE_MSG_CHUNK_COUNT=60
+CONFIG_BT_MESH_TX_SEG_MSG_COUNT=2
+CONFIG_BT_MESH_ACCESS_DELAYABLE_MSG_COUNT=6
```

Comment:
Does this really solve anything? As far as I could tell, it was the number of chunks that ran out, not the number of messages?

Reply:
Fixing the first issue caused new error/wrn message output.

Reply:
Ahh right because it ran out of chunks and then never got to the point where it ran out of messages. Makes sense.